.vscode/settings.json

```diff
@@ -41,5 +41,8 @@
   "eslint.format.enable": false,
   "typescript.tsdk": "node_modules/typescript/lib",
   "jest.jestCommandLine": "node_modules/.bin/jest",
+  "jest.autoRun": {
+    "onStartup": []
+  },
```

Comment:
mine is: `  "jest.autoRun": "off",`

Reply:
Cool I'll fix that. I think best to check it into the repo